src/extension.ts

```diff
@@ -468,9 +470,15 @@ async function explain(obj, field) {
             swaggerSpecPromise = explainer.readSwagger();
         }
 
-        swaggerSpecPromise.then((s) => {
-            resolve(explainer.readExplanation(s, ref));
-        });
+        swaggerSpecPromise.then(
+            (s) => {
+                resolve(explainer.readExplanation(s, ref));
+                vscode.window.showErrorMessage('in after swagger spec promise');
```

Comment:
pretty sure this needs to be removed

Reply:
haha, my "debug with prompt" style is revealed...

Removed.

Reply:
Let he who has not left an `alert(`value of foo is ${foo}`);` in a production commit cast the first stone...